TestRaftClusterMultipleRestart is timeout

## Flaky Test

### Which jobs are failing
```
panic: test timed out after 5m0s
	running tests:
		TestRaftClusterMultipleRestart (5m0s)
```

### CI link
https://github.com/tikv/pd/actions/runs/11775311359/job/32795518792

### Reason for failure (if possible)


### Anything else

--- a/tests/server/cluster/cluster_test.go
+++ b/tests/server/cluster/cluster_test.go
@@ -589,7 +589,9 @@ func TestRaftClusterMultipleRestart(t *testing.T) {
 	re := require.New(t)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	tc, err := tests.NewTestCluster(ctx, 1)
+	tc, err := tests.NewTestCluster(ctx, 1, func(conf *config.Config, _ string) {
+		conf.LeaderLease = 300
+	})
 	defer tc.Destroy()
 	re.NoError(err)
 

